<?xml version="1.0"?>

<launch>
  <!-- face detection node (detects faces in color image and publishes their positions) -->
  <node name="tracking_evaluator" pkg="cob_people_detection" ns="/cob_people_detection" type="tracking_evaluator" output="screen" >
    <remap from="recognition_msg" to="/cob_people_detection/face_recognition/face_recognitions"/>
    <remap from="tracking_msg" to="/cob_people_detection/detection_tracker/face_position_array"/>
    <param name="person" type="string" value="Richard"/>
  </node>

</launch>
